src/livekit/options.ts

```diff
@@ -23,12 +23,16 @@ const defaultLiveKitPublishOptions: TrackPublishDefaults = {
   red: false,
   forceStereo: false,
   simulcast: true,
-  videoSimulcastLayers: [VideoPresets.h180, VideoPresets.h360] as VideoPreset[],
+  videoSimulcastLayers: [
+    VideoPresets.h180,
+    VideoPresets.h360,
+    VideoPresets.h540,
+  ] as VideoPreset[],
```

Comment:
I might be misunderstanding the livekit docs. But can we actually hand it three presets here? Livekit docs stated up to 2 custom layers.
```
    videoSimulcastLayers: [VideoPreset]
    custom video simulcast layers for camera tracks, defaults to h180, h360, h540 You can specify up to two custom layers that will be used instead of the LiveKit default layers. Note: the layers need to be ordered from lowest to highest quality
```